data/description_text.yaml

```diff
@@ -18,19 +18,11 @@ description_text-markdown: |+
 directions_text-markdown: |+
   ## How to use the map:
   
-  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
-  tempor incididunt ut labore et dolore magna aliqua. Consequat ac felis
-  donec et odio pellentesque. Sapien faucibus et molestie ac feugiat. 
-  Morbi tristique senectus et netus et malesuada fames ac. Tempus urna 
-  et pharetra pharetra massa massa ultricies mi quis. Risus nec feugiat 
-  in fermentum. Condimentum mattis pellentesque id nibh tortor id aliquet. 
-  Tortor dignissim convallis aenean et tortor at risus. Praesent elementum
-  facilisis leo vel fringilla. Adipiscing tristique risus nec feugiat in 
-  fermentum. Lacus laoreet non curabitur gravida arcu ac. Rutrum tellus 
-  pellentesque eu tincidunt tortor aliquam nulla. Massa eget egestas purus 
-  viverra accumsan in nisl nisi scelerisque. Faucibus turpis in eu mi 
-  bibendum neque egestas congue quisque. Purus sit amet luctus venenatis 
-  lectus magna fringilla urna. Odio ut enim blandit volutpat maecenas 
-  volutpat. Aliquam sem et tortor consequat id porta. Nunc lobortis mattis 
-  aliquam faucibus. Odio pellentesque diam volutpat commodo sed. Arcu dictum
-  varius duis at consectetur lorem donec massa sapien.
+  Explore how water is used throughout the United States. Click or 
+  hover over on any of the specific use-case boxes below the map to 
```

Comment:
I see where you're going with simplifying it, but it might be best to just say "category" or "Click or hover on the types of water use below the map"

Reply:
still want that changed?